config/prod/ecmonsen-emorypipeline-infra.yaml

```diff
@@ -0,0 +1,22 @@
+template_path: "remote/ecmonsen-emorypipeline-infra.yaml"
+stack_name: "ecmonsen-emorypipeline-infra"
```

Comment:
the purpose of this is to deploy your lambda in Sage-Bionetworks/ecmonsen-emorypipeline-lambda-code` repo correct?  Maybe it should be named `ecmonsen-emorypipeline-lambda` ?

Reply:
That's fine... I changed the stack name and updated filenames to match.

Reply:
Note that I've had to update the filenames in [ecmonsen-emorypipeline-lambda-code](https://github.com/Sage-Bionetworks/ecmonsen-emorypipeline-lambda-code) to match.